### DISCUSSION TITLE
nightfox.lua + lazy.lua in nvim

### DISCUSSION THREAD

**Discussion by robertmonka** [EdenEast/nightfox.nvim#381] [Q&A]
Hi.
In Lazy operating window (:Lazy) I can not see difference between Not selected button and selected button. They both have the same color. For example Ctrl + I for Install button.

There is no difference between "cursor line" and "text section", also here the same color

How could we correct it?

Thanks

**Comment by EdenEast** [comment]
You are correct it looks like lazy is using `CursorLine` and `Visual` for the buttons. These are the same in nightfox. I have created a pr #383 to address this as well as the dimmed and prop highlight groups.
